### PR TITLE
update nvme-cli to 1.6

### PR DESCRIPTION
**Pull request by yuval-lb**
Issue:LBM1-6575

### CODE REVIEW

File: docker/build-inaugurator.dockerfile [hunk 1]
```diff
@@ -39,6 +38,8 @@ WORKDIR /root
 CMD make -C osmosis build -j 10 && \
     make -C osmosis egg
 
+RUN rpm -i https://rpmfind.net/linux/fedora/linux/releases/29/Everything/x86_64/os/Packages/n/nvme-cli-1.6-1.fc29.x86_64.rpm
```
Comment (sashas-lb) [inline docker/build-inaugurator.dockerfile]: i think it is a bad idea .. it will become eventually unavailable ..

**Comment by yuval-lb** [comment]
@efrat-lb I first tried to move to fedora29 but it breaks the makefile and cant compile osmosis.

File: docker/build-inaugurator.dockerfile (hunk 1 above)
Comment (yuval-lb) [reply docker/build-inaugurator.dockerfile]: what you suggest?

Comment (sashas-lb) [reply docker/build-inaugurator.dockerfile]: dont have any suggestion. maybe take our own nvme cli that we have

Comment (yuval-lb) [reply docker/build-inaugurator.dockerfile]: I currently merging this and open a ticket to change to our own nvme-cli

Comment (sashas-lb) [reply docker/build-inaugurator.dockerfile]: @yuval-lb  i though our policy is to get an approval before we merge :)

Comment (yuval-lb) [reply docker/build-inaugurator.dockerfile]: :+1:

**Pull request merged**